GOVERNANCE.md

```diff
@@ -1,52 +1,45 @@
-
-- nominee's first and last name,
-- nominee's email address and GitHub user name,
-- an explanation of why the nominee should be a maintainer,
-- a list of links to non-trivial pull requests (top 10) authored by the nominee.
-
-Maintainers can be removed by a 2/3 majority vote.
+- First decide whether your organization really needs more people with maintainer access. Valid reasons are "blast radius", a large organization that is working on multiple unrelated projects, etc.
+- Contact a sponsor for your organization and express interest.
+- Start doing PRs and code reviews under the guidance of your maintainer.
+- After a period of 1-2 months the existing sponsors will discuss granting maintainer access.
+- Maintainer access can be upgraded to sponsor access after another conference of the existing sponsors.
 
-## Approving PRs
+# Maintainer responsibilities
 
-PRs may be merged after receiving at least one approval from a maintainer.
+- Monitor email aliases.
+- Monitor Discord (delayed response is perfectly acceptable).
+- Triage GitHub issues and perform pull request reviews for other maintainers and the community.
+- Triage build issues - file issues for known flaky builds or bugs, and either fix or find someone to fix any master build breakages.
+- During GitHub issue triage, apply all applicable ([labels](https://github.com/backstage/backstage/labels)) to each new issue. Labels are extremely useful for future issue follow up. Which labels to apply is somewhat subjective so just use your best judgment. A few of the most important labels that are not self explanatory are:
+  - good first issue: Mark any issue that can reasonably be accomplished by a new contributor with this label.
+  - help wanted: Unless it is immediately obvious that someone is going to work on an issue (and if so assign it), mark it help wanted.
+- Make sure that ongoing PRs are moving forward at the right pace or closing them.
+- Participate when called upon in the security release process. Note that although this should be a rare occurrence, if a serious vulnerability is found, the process may take up to several full days of work to implement. This reality should be taken into account when discussing time commitment obligations with employers.
+- In general, continue to be willing to spend at least 25% of one's time working on Backstage (~1.25 business days per week).
+- We currently maintain an "on-call" rotation within the maintainers. Each on-call is 1 week. Although all maintainers are welcome to perform all of the above tasks, it is the on-call maintainer's responsibility to triage incoming issues/questions and marshal ongoing work forward. To reiterate, it is not the responsibility of the on-call maintainer to answer all questions and do all reviews, but it is their responsibility to make sure that everything is being actively covered by someone.
```

Comment:
Happy to see these written objectively. :)

I wonder if a word about support for plugins can be added here? e.g. "Have eyes on PRs from plugin developers, suggest improvements, possible use of APIs from core, etc.". Things you already do!

I also see this section growing out in the future, but looks good as of now.